tests/publiccloud/slem_basic.pm

```diff
@@ -39,12 +40,48 @@ sub check_avc {
     }
 }
 
+sub check_selinux_enforced {
+    # Check SELinux-enforced VMs on publiccloud
+    my ($self) = @_;
+    my $log_avc = "selinux_avc_audit.txt";
+    my $ssh_opts = "-t -o ControlPath=none";
+
+    my $instance = $self->{my_instance};
+    my $out;
+
+    $out = $instance->ssh_script_output("cat /sys/kernel/security/lsm", ssh_opts => $ssh_opts);
+    die("unexpected SELinux status in kernel security lsm: $out") unless (grep(/^selinux$/, split(/,/, $out)));
+    record_info("SELinux status", $out);
+
+    $out = $instance->ssh_script_output("sestatus", ssh_opts => $ssh_opts);
+    die("Unexpected SELinux status: $out") unless (grep(/Current mode:\s+enforcing/, split(/\n+/, $out)));
+    record_info("SELinux status", $out);
+
+    # Check audit status:
+    $instance->ssh_assert_script_run("sudo systemctl is-active auditd", ssh_opts => $ssh_opts);
+    $instance->ssh_assert_script_run("sudo [ -s /var/log/audit/audit.log ]", ssh_opts => $ssh_opts);
+    $instance->ssh_assert_script_run("sudo which ausearch", ssh_opts => $ssh_opts);
```

Comment:
This test is redundant because you run this command immediately afterwards.  If the command doesn't exist, the next command will fail. 

```suggestion
```